source/_docs/relaunch.md

```diff
@@ -1,84 +1,57 @@
 ---
-title: Switching DNS From One Pantheon Site to Another
-description: Learn how to change DNS details between Pantheon Drupal or WordPress sites.
+title: Relaunch Existing Pantheon Site
+description: Take a new site live by moving custom domains from one Site Dashboard to another, with minimal HTTPS interruptions.
```

Comment:
@ari-gold based on what I observed during testing (described in related issue), I intentionally phrased this as "minimal HTTPS interruptions." Was not confident in saying no interruption